tools/convert_cr_to_oc.py

```diff
@@ -21,21 +22,21 @@ def convertname(root, files, name):
 		os.unlink(os.path.join(root, "Names.txt"))
 		r = "\"$Name$\""
 	else:
-		print root, name
-	for lang, localname in names.iteritems():
+		print(root, name)
+	for lang, localname in names.items():
 		if "StringTbl" + lang + ".txt" in files:
-			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"rb").read().splitlines()
+			lines = open(os.path.join(root, "StringTbl" + lang + ".txt"),"r").read().splitlines()
 		else:
 			lines = ()
-		f = open(os.path.join(root, "StringTbl" + lang + ".txt"),"wb")
+		f = open(os.path.join(root, "StringTbl" + lang + ".txt"),"w")
 		for line in lines:
 			f.write(line)
 			f.write("\r\n")
```

Comment:
...in particular in places like this, where the CRT translation layer will insert an additional `\r` in front of the `\n`, so you'd end up with `\r\r\n`.